pkg/validation/validation_test.go

```diff
@@ -146,3 +151,114 @@ func TestContainerPorts(t *testing.T) {
 		}
 	}
 }
+
+func TestNodePorts(t *testing.T) {
+	httpPort := int32(30081)
+	httpsPort := int32(30444)
+
+	testCases := []struct {
```

Comment:
possibly need a test case for invalid port names if we don't use kubebuilder validation